README.md

```diff
@@ -1,4 +1,20 @@
 Put the time in weeks.  We'll change the format as need be.  I'm too busy
 putting my kid to sleep to do this nicely ;)
 
+** Current Format Requirements **
+Column and Type:
+
+* Company [string] - Company Name Or Governing Body
+* Paternity [int] - Number of Weeks Paid Paternity Leave
+* Maternity [int] - Number of Weeks Paid Maternity Leave
+* Adoption [int] - Number of Weeks Paid Leave Following an Adoption
+* Notes [string] - Any additional information of note
+* Source [string] - Document Source if Available 
+* Paid/Unpaid [string] - Either 'Paid' or 'Unpaid' as appropriate
+* Policy_Creation_Date [date] - Date of Creation (year, include month and day if possible)
+* Policy_Last_Updated [date] - Date of Last Policy Update (year, include month and day if possible)
+* Date_Added_to_Repo [date] - Date Stamp the record was added to this registry.
```

Comment:
So I've been thinking about this... and we really have this data in Github, and if you wanted to ingest this into ES we could script extracting the year.  That leaves this list as a point in time snapshot for people to look at and not have to sift through the latest entry for a company and wonder if it's up to date (we'll just assume that it always is up to date).